buildpacks/jvm/src/openjdk_version.rs

```diff
@@ -0,0 +1,338 @@
+use crate::util::zip_longest;
+use nom::Finish;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::cmp::Ordering;
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+#[derive(Clone, Debug)]
+#[allow(dead_code)]
```

Comment:
(optional) It looks like this `#[allow(dead_code)]` annotation was added for the `suffix` field? Would it make sense to move the annotation to that specific field rather than applying it to the whole enum, to prevent masking future unrelated dead code? Or given this is an internal type, it seems like the unused `suffix` field could be removed entirely, or else renamed to `_suffix` (which wouldn't need a `#[allow(dead_code)]` annotation)?

Reply:
I opted to localize the `#[allow(dead_code)]` a little more. I plan to use this parser and types in other JVM related tooling soon. It will then move into its own crate where the concerns will just go away as the suffix field will be 
`pub`.